examples/tictactoe/ts/src/run.ts

```diff
@@ -10,122 +10,101 @@
-  const zkAppInstance = new TicTacToe(zkAppPubkey);
-
-  // Create a new instance of the contract
-  console.log('\n\n====== DEPLOYING ======\n\n');
-  await deploy(zkAppInstance, zkAppPrivkey, player1);
-
-  console.log('after transaction');
-
-  // initial state
-  let b = await Mina.getAccount(zkAppPubkey);
-
-  console.log('initial state of the zkApp');
-  for (const i in [0, 1, 2, 3, 4, 5, 6, 7]) {
-    console.log('state', i, ':', b.appState?.[i].toString());
-  }
-
-  console.log('\ninitial board');
-  new Board(b.appState?.[0]!).printState();
-
-  // play
-  console.log('\n\n====== FIRST MOVE ======\n\n');
-  await makeMove(
-    zkAppInstance,
-    zkAppPrivkey,
-    player1,
-    player1Public,
-    player2Public,
-    Field.zero,
-    Field.zero
-  );
-
-  // debug
-  b = await Mina.getAccount(zkAppPubkey);
-  new Board(b.appState?.[0]!).printState();
-
-  // play
-  console.log('\n\n====== SECOND MOVE ======\n\n');
-  await makeMove(
-    zkAppInstance,
-    zkAppPrivkey,
-    player2,
-    player1Public,
-    player2Public,
-    Field.one,
-    Field.zero
-  );
-  // debug
-  b = await Mina.getAccount(zkAppPubkey);
-  new Board(b.appState?.[0]!).printState();
-
-  // play
-  console.log('\n\n====== THIRD MOVE ======\n\n');
-  await makeMove(
-    zkAppInstance,
-    zkAppPrivkey,
-    player1,
-    player1Public,
-    player2Public,
-    Field.one,
-    Field.one
-  );
-  // debug
-  b = await Mina.getAccount(zkAppPubkey);
-  new Board(b.appState?.[0]!).printState();
-
-  // play
-  console.log('\n\n====== FOURTH MOVE ======\n\n');
-  await makeMove(
-    zkAppInstance,
-    zkAppPrivkey,
-    player2,
-    player1Public,
-    player2Public,
-    Field(2),
-    Field.one
-  );
-
-  // debug
-  b = await Mina.getAccount(zkAppPubkey);
-  new Board(b.appState?.[0]!).printState();
-
-  // play
-  console.log('\n\n====== FIFTH MOVE ======\n\n');
-  await makeMove(
-    zkAppInstance,
-    zkAppPrivkey,
-    player1,
-    player1Public,
-    player2Public,
-    Field(2),
-    Field(2)
-  );
-
-  // debug
-  b = await Mina.getAccount(zkAppPubkey);
-  new Board(b.appState?.[0]!).printState();
-  console.log(
-    'did someone win?',
-    b.appState?.[2].toString() ? 'Player 1!' : 'Player 2!'
-  );
-
-  // cleanup
-  await shutdown();
+await isReady;
+let Local = Mina.LocalBlockchain({ proofsEnabled: false });
+Mina.setActiveInstance(Local);
+const [{ privateKey: player1 }, { privateKey: player2 }] = Local.testAccounts;
+
+const zkAppPrivateKey = PrivateKey.random();
+const zkAppPublicKey = zkAppPrivateKey.toPublicKey();
+const zkApp = new TicTacToe(zkAppPublicKey);
+
+// Create a new instance of the contract
+console.log('\n\n====== DEPLOYING ======\n\n');
+const txn = await Mina.transaction(player1, () => {
+  AccountUpdate.fundNewAccount(player1);
+  zkApp.deploy();
+  zkApp.startGame(player1.toPublicKey(), player2.toPublicKey());
+});
+await txn.prove();
+await txn.sign([zkAppPrivateKey]).send();
```

Comment:
It might be redundant but I think it would be helpful to add the same comment you did on the Sudoku example about `tx.sign()`